README.md

```diff
@@ -131,6 +142,15 @@ scrape_configs:
         regex: .*
         target_label: __address__
         replacement: 127.0.0.1:9115  # Blackbox exporter.
+
+  - job_name: 'blackbox-post'
+    metrics_path: /probe
+    params:
+      body: ['{}'] # Content, type must match the Content-Type header configured in the module
```

Comment:
May I ask why? It's more flexible having it in the scrape config. We have multiple endpoints that we scrape, at the moment with the same body, but if we needed different body content, we would need to have multiple module declarations and could no longer use generic names such as http_json_post_2xx, we'd need http_json_post_2xx_service_a and http_json_post_2xx_service_b etc. 

Kind regards

Andrew